torch/_inductor/lowering.py

```diff
@@ -585,16 +586,8 @@ def _get_primitive_bitwidth(dtype):
     if src_bits != dst_bits:
```

Comment:
We should be able to handle the conversion between types with different bit_widths now. will double check.

Reply:
removed outdated fallback in https://github.com/pytorch/pytorch/pull/128883/commits/c9e686b67e2724d42249c6d2107e20c6ed13a46f since the new Dtypeview takes controls for data types with different bit_wdiths.